ui/app/templates/components/console/command-input.hbs

```diff
@@ -10,19 +10,20 @@
     <Chevron />
   {{/if}}
   <input
-    aria-label="command input"
+    aria-label="web R.E.P.L."
```

Comment:
Adding it as "R.E.P.L." instead of just REPL helps screen readers to read it out correctly.